### ISSUE TITLE
Screen reader announcement is bit different while interacting with signposts

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4653]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.2 Name, role, value.

**Test Environment:**
- Test URL: https://clarity.design/documentation/signposts
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto any of the signposts and try to interact with it - expand/collapse, move focus in and out, etc. Note: Use arrow key as well as tab navigation.

**Expected result:**
1. When screen reader user moves focus onto the signpost, appropriate label, control type and state should be annnounced.
2. When screen user moves focus in and out of expanded content, announcement should be standard or usual without any verbosity or additional content announcement. 

**Actual result:**
1. Toggle word is announced along with expanded state which is bit verbose. The expanded state conveys the semantics for toggle, so the word "toggle" can be omitted from the label.
2. When focus is moved to the expanded content, the word "clickable" is read along with the heading which can be confusing. When user navigates away navigates backwards towards the expanded content, the word "clickable" is read along with close button. So it can be said that, when user enters the expanded content area, the word clickable is tagged to whatever element is focused.

**Comment by gkhadK** [comment]
Additional observation: No announcement is made after closing the SignPosts popup by pressing close button. If escape key is pressed instead of close button announcement is made as "Collapsed".
